dae/module.nix

```diff
@@ -34,7 +34,7 @@ in
       enable = mkEnableOption "dae, a Linux high-performance transparent proxy solution based on eBPF";
 
       package = mkOption {
```

Comment:
Add `type = lib.types.package;`
I think it's better to specify the type of package. This can expose errors earlier.